aii-core/src/main/perl/aii-shellfe

```diff
@@ -375,6 +375,8 @@ use constant DHCPPATH  => '/system/aii/dhcp';
 use constant MAC       => '--mac';
 
 
+Readonly my $PROTECTED_COMMANDS   => 'remove|configure|install';
+Readonly my $PROTECTED_OPTION   => 'protectid';
```

Comment:
How about unprotectid to make it clearer that this option removes protection? Or --remove-protection-id ?